### ISSUE TITLE
Improve activity log and support activity feed

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#5729]
**Describe the problem you would like to solve**
Every submission has an activity log which can be used to track all editorial activity related to that submission. This activity log is a valuable source of information but is not currently given much consideration -- in how users access it, how useful the information is, and how prominently it is displayed.

**Describe the solution you'd like**
A new activity feed panel that combines the current Tasks panel with a global activity feed which contains log entries for all submissions a user is assigned to as well as . The log entries should be updated to include more helpful information and links, such as:

- A link to the related submission.
- The type of log entry (Discussion, File Upload, Review Assignment, etc)
- Useful information about the entry (ie - for discussions, a snippet of the new message)

**Who is asking for this feature?**
This feature has not been requested. However, I think it would go nicely with our efforts to refactor notifications and emails throughout the applications. For example, it would be nice if eventually users could control which kinds of log entries resulted in email notifications.

**Additional information**
A mockup of an example activity feed panel.
![Selection_009](https://user-images.githubusercontent.com/2306629/79221739-434ead80-7e4e-11ea-9275-6b382db523f9.png)

**Comment by alexxxmendonca** [comment]
This looks... great? Good job!

If possible, please make sure to distinguish when a user performs an action logged as another user.

**Comment by NateWr** [comment]
Good idea, @alexxxmendonca! I should caution that this isn't planned for the near future. But I filed the issue to flesh out some roadmap/planning we are working on.

**Comment by YvesLepidus** [comment]
Hi all!
Is there any plan to resume this activity feed? I think this would be useful for the issue: [OPS - Notify Moderators when a new file has been uploaded](https://github.com/pkp/pkp-lib/issues/5975).

**Comment by asmecher** [comment]
@Devika008, I think you're considering new presentations for the activity log, right? Do you have an open github issue for that (which could be cross-linked here)?

**Comment by Devika008** [comment]
Hi @asmecher 

I've created the design for the activity log, but it doesn't yet include the considerations Nate highlighted. I can revise the design to incorporate his requests and share an updated version here. Let me know if that works! 😊

![Image](https://github.com/user-attachments/assets/c8028369-1779-4693-8638-dbb262eb3be5)

**Comment by Devika008** [comment]
Hello,

Here’s a revised design for the Universal Activity Feed:

![Image](https://github.com/user-attachments/assets/ae24aadf-a36c-46f7-af93-ca30757f1fc5)

Key elements included in the design:

Date – When the action was taken.
Submission – Displays the submission ID and author’s name.
Stage – Indicates the current stage of the submission.
Type – Specifies the type of action, such as a file upload, discussion, or editorial decision.
Event – Describes what happened and who performed the action, including cases where an action was done by someone signed in as another user.
Action – Lists any available actions related to the event.

This is just an initial design, and if everyone likes the overall look and feel, I can expand it to cover all notification edge cases across the system. 😄

**Comment by Tribunal33** [comment]
@Devika008 I had one suggestion, one note, and aone QA request :).  
The Suggestion : Under the submission column if a link to the submission could be included? Would be nice not to have to close down the feed, find the submission then go to the submission to check out the activity.  I was thinking it could be a default action but could be confusing to pu in the action column.
The Note: Noticed in the current task/activity feed is a tab for Notes don't know if this tab/area will go away or not?
The QA Request: Please add any details on how to generate these different logs or users who able to see particular logs,  I wouldl be very helpful to have these requirements for Dev and QA.

**Comment by asmecher** [comment]
@Devika008, if you're ready to update the design work here from your more recent Figma work, please do! Also, let's keep https://github.com/pkp/pkp-lib/issues/5975 separate so we don't lose the notification question in amongst the discussion about logs.
